src/app/workspace/dto/create-workspace.dto.ts

```diff
@@ -1,9 +1,12 @@
 import { ApiProperty } from '@nestjs/swagger';
-import { IsString } from 'class-validator';
+import { IsString, Length } from 'class-validator';
 
 export class CreateWorkspaceDto {
   @ApiProperty()
   @IsString()
+  @Length(1, undefined, {
```

Comment:
This undefined means that we have no boundaries. Does limit have more sense in this context?